### ISSUE TITLE
Keyboard shortcuts for macOS

### ISSUE THREAD

**Issue by tordans** [bryceco/GoMap#746]
Now that the macOS and iOS version [are kept in sync](https://github.com/bryceco/GoMap/issues/654#issuecomment-2078295027) I wanted to test mapping with GoMap on my Mac.

On thing I noticed is, I wanted to press <kbd>w</kbd> to toggle the wireframe mode like I do in iD. That might have be triggered by reading about the [recent move to make the map GoMap and iD look more similar](https://github.com/bryceco/GoMap/pull/734).

But just in case, maybe we can collect some input on possible keyboard shortcuts in macOS GoMap.

**Comment by extratone** [comment]
Can I also request keyboard shortcuts for the iOS app? Or would it be best to open another issue?

**Comment by bryceco** [comment]
You can add them here if you're referring to shortcuts while using a physical external keyboard (with iPad).

**Comment by bryceco** [comment]
> Is there a list of keyboard shortcuts we can use on macOS?

Just the ones I mention above, plus ESC for the tagging popup.

**Comment by gy-mate** [comment]
Cmd—`,` would also be nice for opening the settings menu. It's a regular shortcut used by most macOS apps.

**Comment by bryceco** [comment]
> Cmd—`,` would also be nice for opening the settings menu. It's a regular shortcut used by most macOS apps.

Yeah, that makes sense, but I'm not sure which Settings you mean since we have two:
* The Display Settings (map icon) that relates to how the map appears (kinda)
* App Settings (gear icon) that has a bunch of less commonly used stuff that doesn't really relate to editing.

As a Mac user I would expect Cmd-, to open the App Settings panel, but that is (by design) very rarely used.

**Comment by gy-mate** [comment]
I meant the App Settings. But it would also be great if Display Settings got a different keyboard shortcut, if it’s not too much work.

**Comment by bryceco** [comment]
Command-, support added in 958dc867268cb133aed582f32c7c2918c865e4a8

**Comment by gy-mate** [comment]
Awesome, thanks! :)
